variants/esp32s2/pins_arduino.h

```diff
@@ -3,6 +3,11 @@
 
 #include <stdint.h>
 
+#define USB_VID            0x303A
+#define USB_PID            0x0002
+#define USB_MANUFACTURER   "Espressif Systems"
+#define USB_PRODUCT        "ESP32S2_DEV"
```

Comment:
explicit ESP32S2 dev module USB info so that it could be picked up with user select Adafruit TinyUSB